Modules/Commvault.CommCell/Commvault.CommCell.psm1

```diff
@@ -418,6 +418,250 @@ function Get-CVClient {
+.PARAMETER Force
+    Switch to Force override of default 'WhatIf' confirmation behavior.
+.OUTPUTS
+    Outputs errorMessage and errorCode. Error code 0 is success.
+.NOTES
+    Author: Craig Tolley
+    Based on API Documentation at https://documentation.commvault.com/commvault/v11/article?p=92013.htm
+#>
+    [CmdletBinding(DefaultParameterSetName = 'ByName', SupportsShouldProcess = $True, ConfirmImpact = 'High')]
+    Param (
+        [Alias('Client')]
+        [Parameter(Mandatory = $True, ParameterSetName = 'ByName', ValueFromPipeline = $True, ValueFromPipelineByPropertyName = $True)]
+        [ValidateNotNullorEmpty()]
+        [String] $Name,
+
+        [Alias('ClientId')]
+        [Parameter(Mandatory = $True, ParameterSetName = 'ById', ValueFromPipeline = $True, ValueFromPipelineByPropertyName = $True)]
+        [ValidateNotNullorEmpty()]
+        [Int32] $Id,
+
+        [Parameter(Mandatory = $True, ParameterSetName = 'ByName')]
+        [Parameter(Mandatory = $True, ParameterSetName = 'ById')]
+        [System.Object] $LicensesToRelease,
+
+        [switch]$Force
+    )
+
+    begin { Write-Debug -Message "$($MyInvocation.MyCommand): begin"
+
+        try {
+            $sessionObj = Get-CVSessionDetail $MyInvocation.MyCommand.Name
+            $endpointSave = $sessionObj.requestProps.endpoint
+        }
+        catch {
+            throw $_
+        }
+    }
+
+    process { Write-Debug -Message "$($MyInvocation.MyCommand): process"
+
+        try {
+            $sessionObj.requestProps.endpoint = $endpointSave
+            
+            if ($PSCmdlet.ParameterSetName -eq 'ById' ) {
+                $sessionObj.requestProps.endpoint = $sessionObj.requestProps.endpoint -creplace ('{clientId}', $Id) 
+            }
+            else {
+                $clientObj = Get-CVClient -Name $Name
+                if ($null -eq $clientObj) { 
+                    Write-Information -InformationAction Continue -MessageData "INFO: $($MyInvocation.MyCommand): client not found having name [$Name]"
+                    return
+                }
+                $Id = $clientObj.clientId
+            }
+
+            <#
+            {
+                "isClientLevelOperation": true,
+                "licensesInfo": [{
+                "platformType": 1,
+                "license": {
+                    "appType": 33,
+                    "licenseName": "Server File System - Windows File System"
+                }
+                }],
+                "clientEntity": {
+                "clientId": "client001"
+                }
+            }
+            #>
+
+            $body = @{}
+
+            $client = @{}
+            $client.Add('clientId', $Id)
+            $body.Add('clientEntity', $client)
+            
+            $licensesinfos = @()
+            ForEach ($l in $LicensesToRelease) {
+                $licenseinfo = @{}
+                $licenseinfo.Add("platformType",$LicensesToRelease.platformType)
```

Comment:
The payload here is incorrect platformType is populated with array of integers for multiple licenses . Instead of $LicensesToRelease replace it with the $l.platformType.